augment/transforms.py

```diff
@@ -93,6 +93,55 @@ def __call__(self, m):
         return m
 
 
+class Random_contrast(object):
```

Comment:
Could you try to use camel case for the class names, e.g. `RandomContrast` instead of `Random_contrast` (also for the classes you define below)?